src/plugins/kibana_usage_collection/server/collectors/saved_objects_counts/get_saved_object_counts/get_saved_object_counts.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { estypes } from '@elastic/elasticsearch';
+import { ElasticsearchClient } from '@kbn/core/server';
+
+/**
+ * @private
+ * Lisf of all the active SO Types in Kibana.
+ *
+ * @remarks There is a Jest Integration Test to make sure devs keep this list up-to-date whenever they create/remove their SO types.
```

Comment:
it's not that often a new type gets added, but still frustrating if the build fails because you added a type.

Could we use the saved objects type registry to get a list of registered types? That would mean we would automatically include 3rd party types into the metric. I don't think we _need_ that but it's also not a problem if it gets reported.

Reply:
> That would mean we would automatically include 3rd party types into the metric. I don't think we _need_ that but it's also not a problem if it gets reported.

If we are OK reporting 3rd party types, it will even simplify the aggregations and post-processing logic. 🧡 

Thanks for the comment. I'll update the code accordingly :)